### PR TITLE
add edit profile route

### PR DESCRIPTION
**Pull request by Nazeeh21**

### CODE REVIEW

File: packages/dapp/src/views/editBasicProfile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+          </FormErrorMessage>
+        </FormControl>
+
+        <FormControl>
+          <FormLabel>Description</FormLabel>
+          <Textarea
+            borderColor="purple.500"
+            type={"text"}
+            placeholder="Web3 and Blockchain enthusiast"
+            // value={description}
```
Comment (damianmarti) [inline packages/dapp/src/views/editBasicProfile/index.tsx:233]: Why is it commented?

File: packages/dapp/model.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"definitions":{"basicProfile":"kjzl6cwe1jw145cjbeko9kil8g9bxszjhyde21ob8epxuxkaon1izyqsu8wgcic","cryptoAccounts":"kjzl6cwe1jw149z4rvwzi56mjjukafta30kojzktd9dsrgqdgz4wlnceu59f95f","alsoKnownAs":"kjzl6cwe1jw146zfmqa10a5x1vry6au3t362p44uttz4l0k4hi88o41zplhmxnf","privateProfile":"kjzl6cwe1jw1464bnqysstn5eshsbk9yz2dnbtcfzhbpqwue2k32leeivlcjt5g","publicProfile":"kjzl6cwe1jw147i0qxo4lh6eogtql63pvbrso7sjs0r2fxp86eq811wm81i18z7"},"schemas":{"BasicProfile":"ceramic://k3y52l7qbv1frxt706gqfzmq6cbqdkptzk8uudaryhlkf6ly9vx21hqu4r6k1jqio","CryptoAccounts":"ceramic://k3y52l7qbv1frypussjburqg4fykyyycfu0p9znc75lv2t5cg4xaslhagkd7h7mkg","AlsoKnownAs":"ceramic://k3y52l7qbv1fryojt8n8cw2k04p9wp67ly59iwqs65dejso566fij5wsdrb871yio","PrivateProfile":"ceramic://k3y52l7qbv1frybrrtk1m5oohw0x2b8x90kucjmf1pjkjh2kfiha2i2ybzn6pncow","PublicProfile":"ceramic://k3y52l7qbv1fry3nnnc4t0nwxi42xfvp7k3f7eve8nxhin70si735g1dmq2gr356o"},"tiles":{}}
```
Comment (damianmarti) [inline packages/dapp/model.json:1]: do you need this file? Can we use the one from schema folder?

File: packages/dapp/src/views/editPublicProfile/index.tsx [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+}
+
+const EditPublicProfile = ({
+  nextStep,
+  prevStep,
+  activeStep,
+}: EditPublicProfileProps) => {
+  const { account, contracts, mySelf } = useContext(Web3Context);
+  const { accentColor } = useCustomColor();
+  const [title, setTitle] = useState<string>("");
```
Comment (damianmarti) [inline packages/dapp/src/views/editPublicProfile/index.tsx:40]: This title is not saved on Ceramic.

File: packages/dapp/src/views/editBasicProfile/index.tsx (hunk 1 above)
Comment (Nazeeh21) [reply packages/dapp/src/views/editBasicProfile/index.tsx:233]: yeah, I don't see any point in this. I'll remove it

File: packages/dapp/model.json (hunk 2 above)
Comment (Nazeeh21) [reply packages/dapp/model.json:1]: Yeah right. I'll remove it

File: packages/dapp/src/views/editPublicProfile/index.tsx (hunk 3 above)
Comment (Nazeeh21) [reply packages/dapp/src/views/editPublicProfile/index.tsx:40]: Oh, I see. will fix it soon.

File: packages/dapp/model.json (hunk 2 above)
Comment (Nazeeh21) [reply packages/dapp/model.json:1]: Should I delete this file or update this with the one from the schema folder?

Comment (damianmarti) [reply packages/dapp/model.json:1]: I'm not sure. Do we need this file or we can use the one from the schema folder (updating it)?

Comment (Nazeeh21) [reply packages/dapp/model.json:1]: For now, I'll replace it with the one from the schema. In future, if we found that we don't need it then we'll delete it. Does that sound good?

**Comment by Nazeeh21** [comment]
@damianmarti I've made the suggested changes and resolved conflicts too. Please have a look and provide suggestions

**Review by damianmarti — APPROVED** [review]

**Comment by damianmarti** [comment]
> @damianmarti I've made the suggested changes and resolved conflicts too. Please have a look and provide suggestions

Merged!! Great work!!

**Comment by Nazeeh21** [comment]
Thanks 😄

**Pull request merged**